app/models/goal.py

```diff
@@ -1,6 +1,13 @@
 from flask import current_app
 from app import db
+from sqlalchemy import Table, Column, Integer, ForeignKey
```

Comment:
Hm... I'm not sure you needed these imports as `Column`, `Integer`, and `ForeignKey` should be coming from your own instance of SqlAlchemy called `db`.